nncf/torch/quantization/init_range.py

```diff
@@ -89,12 +91,37 @@ def get_init_config_for_scope_and_group(self, qid: QuantizerId, group: Quantizer
                          "definition!".format(str(qid)))
 
 
+class PTRangeInitCollectorParams(RangeInitCollectorParams):
+    def __init__(self, is_weights: bool, mode: QuantizationMode, per_channel: bool, init_type: str, input_shape: tuple,
+                 channel_idx: int, collectors_with_per_sample_stat_collection: list):
+        super().__init__(is_weights, mode, per_channel, init_type)
+        self._input_shape = input_shape
+        self._channel_idx = channel_idx
+        self._collectors_with_per_sample_stat_collection = collectors_with_per_sample_stat_collection
```

Comment:
The `self._collectors_with_per_sample_stat_collection` looks strange as a parameter in the params struct. I propose you remove it and instead decide whether per-sample collection should be turned on for a given collector explicitly in the factory functions such as `StatCollectorGenerator.generate_stat_collector_for_range_init_config`

Reply:
Updated, thenks!